src/components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/prop-types */
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import "./style.css";
+
+const Carousel = ({ slides }) => {
+  const sliderSettings = {
+    dots: false,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    arrows: false,
+  };
+
+  return (
+    <div className="carousel">
```

Comment:
nice you are doing right but, its not reusable, This comp[onent must receivee children , like 
`<SLider {...settings}>
{children}
</SLider>`